### PR TITLE
Final blog wechat puppet

### PR DESCRIPTION
**Pull request by Roxanne718**
Fixes # .

Make sure these boxes are checked before creating your pull request -- thank you!

- [x] Add Blog Header, including title, author, date.  
    For example:  

    ``` md
    ---
    title: "Your Great Blog Title"
    author: your_github_id
    image: /assets/2020/08-your-blog-slug/teaser.jpg
    ---

    YOUR_FIRST_SENTENCE_SHOULD_BE_A_SUMMARY_FOR_YOUR_POST
    ```

- [x] Keep all filenames & url as lowercase, and use `-` to connect words instead of space. e.g. `2017-10-06-wechat-pc-impactor` instead of `2017-10-06-WeChat PC Impactor`,
- [x] Embed the photo & video before publishing, save all external file to a folder named by `${MONTH}-${YOUR_BLOG_SLUG}/` under `/assets/2020` directory.
- [x] Add teaser image for the blog
- [x] Add your self as an author to the author data file at <https://github.com/wechaty/wechaty.js.org/blob/master/_developers/YOUR_GITHUB_USERNAME.md> with your GitHub username. And please also remember to reference yourself with the Github username in your blog post, so that we can have nice author information besides your blog after we published it!
  For excample:
  ```md
  ---
  name: Jiarui LI (李佳芮)
  site: https://lijiarui.github.io
  bio: 句子互动创始人 & CEO，微软人工智能最具价值专家 (AI MVP)
  avatar: /assets/developers/lijiarui/avatar.png
  email: rui@juzi.bot
  twitter: https://weibo.com/u/2175505900
  ---
  ```
  Here is the example file:
  - lijiarui.md: https://github.com/wechaty/wechaty.github.io/blob/master/_developers/lijiarui.md
  - It will shown like this on wechaty blog: https://wechaty.github.io/developers/lijiarui/

To read more about the contribute guideline, visit: <https://github.com/wechaty/wechaty.js.org#contribute-guideline>

### CODE REVIEW

File: jekyll/_posts/2020-09-28-puppet-work-final.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+- 项目成果：
+	- 运用企业微信提供的API接口完成了消息的加密解密以及跨语言的信息传递；
+	- 完成了全部消息类型（文本信息及媒体信息）的封装；
+	- 简单配置了参数及内部逻辑，可以定时拉取形成“即时沟通”的效果；
+	- 撰写简单的配置说明。
+	
+	
+	
+	- 项目仓库位于: <https://github.com/Sapio-S/wechaty-puppet-work>
+	- live coding视频:{% include iframe.html src="https://work.weixin.qq.com/wework_admin/liveroom/v2/replay?ref=app&roomid=6877185498936770818" %}
```
Comment (lijiarui) [inline jekyll/_posts/2020-09-28-puppet-work-final.md:68]: Are you sure this is your live coding video?

**Review by lijiarui — CHANGES_REQUESTED** [review]
Please change as I request.

**Comment by lijiarui** [comment]
@wj-Mcat  Please help @Roxanne718  posting his blog.

File: jekyll/_posts/2020-09-28-puppet-work-final.md (hunk 1 above)
Comment (wj-Mcat) [reply jekyll/_posts/2020-09-28-puppet-work-final.md:68]: Please upload your live-coding video to bilibili/youtube/... video-stream website. Wating for your new commit.

Comment (YQXie99) [reply jekyll/_posts/2020-09-28-puppet-work-final.md:68]: 这个是企业微信自带的直播功能得到的录屏，原本的链接跳转之后应该可以在企业微信登陆后观看。
上传到bilibili/youtube会比较麻烦，所以之前没传，如果必须上传的话可能还得需要一段时间（下载+转换格式+等待审核）。可能得这周末重新提交了orz

**Pull request closed without merging**